extensions/tmtv/src/utils/hpViewports.ts

```diff
@@ -109,15 +109,6 @@ const ptAXIAL = {
         source: true,
         target: true,
       },
-      {
```

Comment:
Please see this.

I removed it because it was linking the Window and Colormap of the PT row with the Fusion row, 

Reasons are:

PT row doesn't have opacity in HP, but fusion row has opacity in HP
Window  level is so different on PT vs fusion

Try adding them back in and you will see what I mean by the above

Reply:
I think we should make it optional to not sync the colormap, and only do window level for that one, you can pass in the options in cornerstone3D. See 

```
if (options?.syncInvertState && invertStateChanged) {
    tProperties.invert = invert;
  }
```

You can have similar pattern of don't sync colormap for this one